examples/js/effects/VREffect.js

```diff
@@ -233,6 +233,20 @@ THREE.VREffect = function ( renderer, onError ) {
 
 	};
 
+	this.requestAnimationFrame = function ( f ) {
+
+		if ( ! isDeprecatedAPI && vrHMD !== undefined ) {
```

Comment:
No, originally I did have that but the spec actually states the rAF should handle correctly when not presenting, behaving same as window.requestAnimationFrame, so not necessary.